.github/workflows/release.yml

```diff
@@ -0,0 +1,12 @@
+name: release
+on:
+  release:
+    types: ["released"]
```

Comment:
Question: will this get triggered also when we publish alpha/beta releases?

Reply:
I tried to explain this in the [original comment](https://github.com/jetstack/cert-manager/pull/4333#issue-707452966) - if it's not clear there, let me know and I'll change it.

Looking back on that, I'm gonna add another line to make it clearer

Reply:
Yeah it's clear thanks, think I read that with insufficient amount of coffee.